src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -184,6 +184,15 @@ public final class Guardrails implements GuardrailsMBean
                     state -> CONFIG_PROVIDER.getOrCreate(state).getDropKeyspaceEnabled(),
                     "DROP KEYSPACE functionality");
 
+    /**
+     * Guardrail disabling bulk loading of SSTables
+     */
+    public static final EnableFlag bulkLoadEnabled =
+    new EnableFlag("bulk_load_enabled",
+                   null,
```

Comment:
Do you have any suggestions on the reason for this feature?

Reply:
That should be the reason why this guardrail exists, to be displayed in the error message. It should be something of the form `Bulk loading is risky because in X conditions Y might happen`, or something like that.

Reply:
This is different from other features as this is disabling the bulk load streaming feature. There is no client state for this and the error message won't be displayed.

Reply:
With the last changes that use `ensureEnabled` the error message with the reason is included in the message, isn't it?

Reply:
I mean the reason won't be sent back to the "client" because there is not client state. Client will only get the "stream failed" message.

Reply:
The reason for disabling bulk load is solely the decision of the Cassandra operator. In our situation, we want to activate this feature only for users who truly require it.

Reply:
@Runtian you just wrote the message which should be present in an exception thrown when it is disabled :)

`Bulk loading of SSTables was disabled by Cassandra operator. We activate this feature only for users who truly require it to not make nodes potentially unstable.`

Something like that ... 

Is @adelapena cool with that?